<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Модель падения паршютиста</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="gridGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>262</width>
            <height>243</height>
           </size>
          </property>
          <property name="title">
           <string>Параметры модели</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="radius">
             <property name="suffix">
              <string> м</string>
             </property>
             <property name="minimum">
              <double>3.500000000000000</double>
             </property>
             <property name="maximum">
              <double>500.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>h =</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Масса тела:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Конечное время:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="startTime">
             <property name="specialValueText">
              <string/>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string> с</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Начальное время:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="mass">
             <property name="suffix">
              <string> кг</string>
             </property>
             <property name="minimum">
              <double>45.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Радиус парашюта</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="endTime">
             <property name="suffix">
              <string> с</string>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="startSpeed">
             <property name="suffix">
              <string> м/с</string>
             </property>
             <property name="maximum">
              <double>15.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="1" alignment="Qt::AlignLeft">
            <widget class="QDoubleSpinBox" name="step">
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Начальная скорость:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Compute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Решить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="WidgetPlot" name="Plot" native="true">
        <property name="minimumSize">
         <size>
          <width>630</width>
          <height>410</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetPlot</class>
   <extends>QWidget</extends>
   <header>widgetplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
